nilearn/image/__init__.py

```diff
@@ -2,7 +2,10 @@
 
 Like, for example, a (3+)D block of data, and an affine.
 """
+# TODO move copy_img out of nilearn._utils..niimg
 from .._utils.niimg import copy_img
+
+# TODO move concat_niimgs out of nilearn._utils.niimg_conversions
 from .._utils.niimg_conversions import concat_niimgs as concat_imgs
```

Comment:
Tempted to say that things that are part of the user facing API should NOT be in a part of the package that is private.